Packs/ProofpointTAP/Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py

```diff
@@ -52,7 +52,7 @@ def get_fetch_times(last_fetch):
     elif isinstance(last_fetch, datetime):
         times.append(last_fetch.strftime(time_format))
     while now - last_fetch > timedelta(minutes=59):
-        last_fetch += timedelta(minutes=59)
+        last_fetch += timedelta(minutes=58)
```

Comment:
it failed on too short and you are making it shorter?

Reply:
It fails at a 30-second interval, 
and the change I made makes sure the shortest interval that can be is a minute.

Reply:
you shouldn't change it as checks for the longest interval

Reply:
or: add an informative comment on why it's 58 minutes.